p2p/protocol/identify/id.go

```diff
@@ -542,10 +549,37 @@ func (ids *IDService) createBaseIdentifyResponse(
 	av := ids.UserAgent
 	mes.ProtocolVersion = &pv
 	mes.AgentVersion = &av
+	udpNAT, tcpNAT := ids.observedAddrs.getNATDeviceTypes()
+	udpNATPb := toPbNATDeviceTyp(udpNAT)
+	tcpNATPb := toPbNATDeviceTyp(tcpNAT)
+	mes.UdpNATDeviceType = &udpNATPb
+	mes.TcpNATDeviceType = &tcpNATPb
 
 	return mes
 }
 
+func toPbNATDeviceTyp(typ network.NATDeviceType) pb.Identify_NATDeviceType {
```

Comment:
typo in method name; shouldn't this be Type?
Also, typ is a very funky variable name, better call it just `t`.

Reply:
done.